### PR TITLE
Add new C# Sample apps of May 2019

### PR DESCRIPTION
**Pull request by sgchoi5**
Added new C# Sample apps for Wearable

### CODE REVIEW

File: docs/application/dotnet/samples/location/overview.md [hunk 1]
```diff
@@ -57,6 +57,15 @@ for TD:
 <div class="tabcontent" id="Wearable">
 <table>
 	<tbody>
+		<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w68findplace.png" width="180" /></p>
+			</td>
+			<td>
+                        <p><strong>(W) FindPlace</strong> [In progress]</p>
+			<p>This sample application demonstrates how to find a place using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a>.</p> and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/location/overview.md]: Fixed it

File: docs/application/dotnet/samples/multimedia/overview.md [hunk 2]
```diff
@@ -113,6 +113,13 @@ for TD:
 <div class="tabcontent" id="Wearable">
 <table>
 	<tbody>
+                <tr>
+                        <td><img alt="" height="180" src="media/w69circularuimediaplayer.png" width="180"/></td>
+                        <td>
+                        <p><strong>(W) CircularUIMediaPlayer</strong> [In progress]</p>
+                        <p>This sample application demonstrates how to build a video player for Tizen Wearables. You can get detailed information from <a href="https://samsung.github.io/Tizen.NET/wearables/CircularUI-MediaPlayer" target="_blank">Tizen .NET Portal</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/multimedia/overview.md]: Fixed it

File: docs/application/dotnet/samples/system/overview.md [hunk 3]
```diff
@@ -85,6 +85,13 @@ for TD:
 			<p>This sample application demonstrates how to play sound and vibration feedback and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/w70gesturesensor.png" width="180"/></td>
+			<td>
+			<p><strong>(W) GestureSensor</strong> [In progress]</p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor</a> and how to detect gestures, such us face down, pick up and wrist up.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/system/overview.md]: Fixed it

File: docs/application/dotnet/samples/location/overview.md [hunk 4]
```diff
@@ -57,6 +57,15 @@ for TD:
 <div class="tabcontent" id="Wearable">
 <table>
 	<tbody>
+		<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w68findplace.png" width="180" /></p>
+			</td>
+			<td>
+                        <p><strong>(W) FindPlace</strong> [In progress]</p>
+			<p>This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a>.</p> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```
Comment (divyk) [inline docs/application/dotnet/samples/location/overview.md]: Please make this small change, as I see a small gap between the lines and there is a period in between the sentence. I have removed the period and /P
Thanks in advance.

This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.

Comment (sgchoi5) [reply docs/application/dotnet/samples/location/overview.md]: Fixed it.

File: docs/application/dotnet/samples/location/overview.md [hunk 5]
```diff
@@ -57,6 +57,15 @@ for TD:
 <div class="tabcontent" id="Wearable">
 <table>
 	<tbody>
+		<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w68findplace.png" width="180" /></p>
+			</td>
+			<td>
+                        <p><strong>(W) FindPlace</strong> [In progress]</p>
+			<p>This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a></p> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```
Comment (divyk) [inline docs/application/dotnet/samples/location/overview.md]: Sorry to say, there is still a break in the line, we have to remove /p, can you please paste the line below to remove the breakage:

This sample application demonstrates how to search for places within a specified area using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Maps.html" target="_blank">Tizen.Maps</a> and how to create a circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.

Comment (sgchoi5) [reply docs/application/dotnet/samples/location/overview.md]: Soryy, I misunderstood your comments. Removed </P>.

**Pull request merged**